blacklistfilter/evaluator/split_evidence/FileHandler.cpp

```diff
@@ -0,0 +1,131 @@
+#include "FileHandler.h"
+#include <fstream>
+#include <iostream>
+#include <cstring>
+#include <sys/stat.h>
+
+FileHandler::TFileWrap::TFileWrap(const std::string& filename, urcsv_t* csv ) 
+: m_filename( filename ),
+m_buffer_len( 0 ) {
+
+    struct stat stat_struct;
+    if( stat(filename.c_str(), &stat_struct) != 0 ) {
```

Comment:
This is not according to coding style. See https://nemea.liberouter.org/developer/coding-style-c